app/models/instance.rb

```diff
@@ -81,6 +81,13 @@ def default?
     host == Instance::DEFAULT_HOST_NAME
   end
 
+  # Returns the full host url.
+  #
+  # @return [String] The full host url with http prefix.
+  def full_url
+    "http://#{host}"
```

Comment:
I don't think it belongs in the model. We do not store the protocol that the user access the site with. It might be HTTP/HTTPS -- and we don't know.

I think for now we should just add the protocol in the view. If we decide to store a full URL in future we'll think about how to deal with this.